apis/v1alpha1/httproute_types.go

```diff
@@ -500,6 +500,16 @@ type HTTPRouteForwardTo struct {
 	// The gateway status for this route should be updated with a
 	// condition that describes the error more specifically.
 	//
+	// The protocol to use is defined using `AppProtocol` field (introduced
+	// in Kubernetes 1.18) in the Service resource. In absence of `AppProtocol` field
+	// (Kubernetes versions < 1.18), `networking.x-k8s.io/app-protocol` annotation
```

Comment:
Since you just specified the Kubernetes version above, I don't think it's needed again.
```suggestion
	// a `networking.x-k8s.io/app-protocol` annotation
```